From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pierre=20Sandstro=CC=88m?= <pierre.sandstrom@dstny.com>
Date: Thu, 15 Dec 2022 17:19:39 +0100
Subject: Adding dependencies for external codecs

This patch adds our external codecs (AMR-WB/NB and G.729) as dependencies to WebRTC.

diff --git a/third_party/webrtc_overrides/BUILD.gn b/third_party/webrtc_overrides/BUILD.gn
index 70010439af41f56e20db6b32f4158b64cef1e0bb..8c20776f006e5b149d888393059daef7230e7f6f 100644
--- a/third_party/webrtc_overrides/BUILD.gn
+++ b/third_party/webrtc_overrides/BUILD.gn
@@ -115,6 +115,8 @@ webrtc_public_deps = [
   "//third_party/webrtc/stats:rtc_stats",
   "//third_party/webrtc/stats:rtc_stats_test_utils",
   "//third_party/webrtc/system_wrappers",
+  "//third_party/webrtc/mediaengine:amr",
+  "//third_party/webrtc/mediaengine:g729",
 ]
 if (defined(rtc_exclude_system_time) && rtc_exclude_system_time) {
   webrtc_public_deps += [ ":system_time" ]
